### PR TITLE
Add initial version of MAINTAINERS file

### PR DESCRIPTION
**Pull request by ret2libc**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [ ] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [x] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

This is a first effort in trying to split the review process to multiple people that should be specialized in some particular areas of the project. This will ensure higher review quality and more distributed process. Please, everybody listed here should give his approval to 1) being listed in the file 2) being the point of reference for a given part of the project. This means that you will be asked to review those parts when there are PRs that change those files/dirs.

@radare I listed you in many places for now, sorry.
Everybody, please, try to look at the various parts of the project and tell me if you want to get involved in more parts (or less, though we hope it will be more! :D)

**Test plan**

We will see how it goes only by using it.

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+libr/anal/xrefs.c: sirmy15@gmail
+libr/arch/gb: condret@runas-racer.com
+libr/asm: pancake@nopcode.org
+libr/asm/p/x86_*: sirmy15@gmail.com
+libr/asm/p/arm_*: pancake@nopcode.org
+libr/bin: sirmy15@gmail.com, pancake@nopcode.org
+libr/bin/format/elf: sirmy15@gmail.com, kazarmy@gmail.com
+libr/bin/format/mach0: pancake@nopcode.org
+libr/bin/format/objc: pancake@nopcode.org
+libr/bin/format/pe:
```
Comment (ret2libc) [inline MAINTAINERS:12]: who is our "expert" for PE files? @pelijah ? Somebody else?

File: MAINTAINERS [hunk 2]
```diff
@@ -0,0 +1,29 @@
+libr/anal: info@florianmaerkl.de
+libr/anal/esil: pancake@nopcode.org, condret@runas-racer.com
+libr/anal/xrefs.c: sirmy15@gmail
+libr/arch/gb: condret@runas-racer.com
+libr/asm: pancake@nopcode.org
```
Comment (wargio) [inline MAINTAINERS:5]: add me also for `libr/asm/p/*` wargio@libero.it

**Comment by radare** [comment]
is this info from git log or copyright headers?

**Comment by ret2libc** [comment]
It does not come from either of those. It's a way for people to know who to contact in case they need help/review for some parts of the code. For example, I am not listed almost anywhere in the "copyright", but I am available and knowledgeable to review some parts of the project.

- copyright info: they mostly list only you, but that does not mean you are the only one that know those parts of the code. So they are not useful to help people understand the points of contact and to split the review work properly.
- git log info: for any given file there is way too many commits and maybe there are many commits that are just crash fixes or simple things and that does not mean that a person is familiar enough with the code. Again, this is not really helpful to understand who to contact in case you need help for a given part of the code. For example, I did not touch too much ELF code lately, but I can for sure help review PRs and I (think) understand the code.

This file would not be something you update every day, so I don't see too much burden in having it. The fact that someone touches everything does not make him automatically knowledgeable enough to review things (or, at least, not to be the main contact). If we know all everything (which I doubt) that would be even better as we can split the work even more.

**Comment by thestr4ng3r** [comment]
I don't think it should be e-mail addresses in there. It's hard to associate them to the github users and I get enough e-mail spam already.

**Comment by ret2libc** [comment]
> I don't think it should be e-mail addresses in there. It's hard to associate them to the github users and I get enough e-mail spam already.

GH nicknames?

**Comment by radare** [comment]
ok, rebase

**Comment by radare** [comment]
ping

**Comment by mohe2015** [comment]
Maybe use https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners instead?

**Comment by ret2libc** [comment]
> Maybe use https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners instead?

Interesting I will have a look! Thanks

**Comment by ret2libc** [comment]
@XVilka @radare I have moved to CODEOWNERS file, which looks awesome and exactly what we want! I'm not sure whether an admin has to do anything to really start using this file, but once that's done it should automatically request reviews to the right people whenever you open a PR. We can easily extend this file.

**Comment by ret2libc** [comment]
@yossizap done! thank you for being available to monitor/review those parts :)

**Comment by ret2libc** [comment]
so @radare are you satisfied with this? Can we merge it?

**Comment by ret2libc** [comment]
> i dont see who is in charge of maintaining the MAINTAINERS file :D

doh! you are right i guess :D

**Pull request merged**